commit 54748e6d47ec78ef7ff1182dd6e0e9aa2fcaed42
Author: Kyösti Mälkki <kyosti.malkki@gmail.com>
Date:   Tue Jul 17 07:04:36 2018 +0300

    Hack fstat()
    
    Python 'import' does not work for paths, i.e.
    sys.path = ['/boot/python'] fails to load any
    .pyc and .py files or recurse into module
    directories. For now, work around the issue
    to get Python env operational.

diff --git a/Python/import.c b/Python/import.c
index e47ce63..460c9ac 100644
--- a/Python/import.c
+++ b/Python/import.c
@@ -129,8 +129,12 @@ static int isdir(char *path) {
 #else
 #ifdef HAVE_STAT
 static int isdir(char *path) {
+#ifndef GRUB_MACHINE_COREBOOT
     struct stat statbuf;
     return stat(path, &statbuf) == 0 && S_ISDIR(statbuf.st_mode);
+#else
+    return (strstr(path, "/boot/python") == path);
+#endif
 }
 #else
 #ifdef RISCOS
